### ISSUE TITLE
Re-initiate tests and implement continuous integration

### ISSUE THREAD

**Issue by zkamvar** [annecori/EpiEstim#64]
Related to #40, 

The new version of EpiEstim currently has no tests and that's.... not good. In fact, with the current master branch, Example 2 fails. 

Regarding tests, the current setup is relatively reasonable since they do not rely on randomization to generate the data, but we need to find out why Example 2 is no good.

This could have been caught earlier with continuous integration, so I would suggest to use the following to create it.

```r
usethis::use_travis()
usethis::use_appveyor()
```

**Comment by jstockwin** [comment]
We do have CI set up, via `.travis.yml`. The way you say is probably much better, though. I'm not so used to R.

You can see the builds here: https://travis-ci.org/annecori/EpiEstim

**Comment by jstockwin** [comment]
Also note that tests were completely disabled in the new version PR. I've added a comment that they should be re-implemented. I'm not sure why they were removed - I guess they were failing..

Ref https://github.com/annecori/EpiEstim/pull/61#pullrequestreview-126694218

**Comment by zkamvar** [comment]
> We do have CI set up, via `.travis.yml`. The way you say is probably much better, though. I'm not so used to R.
> 
> You can see the builds here: https://travis-ci.org/annecori/EpiEstim

Ah! I did not see this :man_facepalming: The `usethis::use_travis()` construct creates the travis file as you currently have it, though it would be good to implement travis so it covers what cran covers and assesses test coverage (when we reinstate those) https://github.com/reconhub/incidence/blob/master/.travis.yml 



> Also note that tests were completely disabled in the new version PR. I've added a comment that they should be re-implemented. I'm not sure why they were removed - I guess they were failing..
> 
> Ref [#61 (review)](https://github.com/annecori/EpiEstim/pull/61#pullrequestreview-126694218)

It looks like they were removed in 5922bd8 and 9a13d55, but there's no explanation as to the reason why they were removed (but now we know who to blame :stuck_out_tongue:)

**Comment by zkamvar** [comment]
Update: I just moved the tests from the master branch to the new-version branch and it appears that only two things needed to change to get the tests working:

1. estimate_r changed to estimate_R 
2. plot = FALSE was no longer a parameter
3. The new output was a class "estimate_R", but the saved results weren't

Example 2, however fails as I mentioned earlier and it appears to be due to the sampling bug that was fixed in R 3.6. Since this is a trivial sampling, I'm going to hard-code the values to avoid having to set or reset the default sampler.
